### PR TITLE
Feature/security-policy

### PR DESCRIPTION
**Pull request by elglup**
## Related issue

#799 

## Overview

Create initial security policy for the standards-toolkit repo

## Link to preview
https://github.com/Royal-Navy/standards-toolkit/blob/feature/security-policy/SECURITY.md

## Work carried out

Created SECURITY.md

### CODE REVIEW

**Review by m7kvqbe1 — COMMENTED** [review]
Looks great to me so far - I think the references to 'Nelson Standards' need to change to 'Navy Digital Standards' based on DTOs Slack comments.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Security Policy
+The Royal Navy Standards security policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
```
Comment (thyhjwb6) [inline SECURITY.md]: is `security policy` a noun and therefore should be `Security Policy`?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The Royal Navy Standards team takes security bugs in the standards-toolkit seriously.  We appreciate your efforts to responsibly disclose your
```
Comment (thyhjwb6) [inline SECURITY.md]: How about `security vulnerabilities` rather than `security bugs`?

Comment (thyhjwb6) [inline SECURITY.md]: `Standards Toolkit` over `standards-toolkit`

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+The below table details which versions of the standards-toolkit are supported with bug fixes and security updates:
```
Comment (thyhjwb6) [inline SECURITY.md]: Same here fore `standards-toolit`

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+| Version | Supported          |
```
Comment (thyhjwb6) [inline SECURITY.md:10]: Can this be simplified so that `1.x` is not supported and `2.x` is? Or something similar...

**Review by thyhjwb6 — COMMENTED** [review]
Looks good, some minor comments 👍

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+The Royal Navy Digital Standards team takes security vulnerabilities in the Standards Toolkit seriously.  We appreciate your efforts to responsibly disclose your
+findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+The below table details which versions of the Standards Toolkit are supported with bug fixes and security updates:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x.    | :white_check_mark: |
```
Comment (thyhjwb6) [inline SECURITY.md]: Does this need the extra `.`?

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# Security Policy
+The Royal Navy Digital Standards Policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
+
+The Royal Navy Digital Standards team takes security vulnerabilities in the Standards Toolkit seriously.  We appreciate your efforts to responsibly disclose your
+findings, and will make every effort to acknowledge your contributions.
```
Comment (thyhjwb6) [inline SECURITY.md]: could this be on the same line as "disclose your"?

**Review by thyhjwb6 — APPROVED** [review]
Looks good, a couple of minor comments.

Are you planning on squashing the commits?

File: SECURITY.md (hunk 5 above)
Comment (elglup) [reply SECURITY.md]: No I'll remove the .

File: SECURITY.md (hunk 6 above)
Comment (elglup) [reply SECURITY.md]: yes done

**Review by thyhjwb6 — APPROVED** [review]
👍 

For future reference it's good if the commit message reads as an imperative instruction eg `Add` over `Added`.

**Pull request merged**